src/main/kotlin/org/wycliffeassociates/otter/common/domain/ResourceContainerConfig.kt

```diff
@@ -0,0 +1,59 @@
+package org.wycliffeassociates.otter.common.domain
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory
+import com.fasterxml.jackson.module.kotlin.KotlinModule
+import org.wycliffeassociates.resourcecontainer.Config
+import java.io.File
+import java.io.IOException
+
+class OtterResourceContainerConfig : Config {
```

Comment:
Rename file to match?

Reply:
thanks, forgot I decided on the different name here